airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -191,8 +196,23 @@ def __init__(
         aws_conn_id: str = "aws_default",
         waiter_countdown: int = 25 * 60,
         waiter_check_interval_seconds: int = 60,
+        # TODO: waiter_max_attempts and waiter_delay should default to None when the other two are deprecated.
+        waiter_max_attempts: int | ArgNotSet = NOTSET,
+        waiter_delay: int | ArgNotSet = NOTSET,
```

Comment:
```suggestion
        waiter_max_attempts: int | None |ArgNotSet = NOTSET,
        waiter_delay: int | None | ArgNotSet = NOTSET,
```

I guess None it is already valid value for this arguments

Reply:
Ah, yep, good catch.  I'll make the change.